pynamodb/tests/test_expressions.py

```diff
@@ -271,6 +271,15 @@ def test_and(self):
         assert placeholder_names == {'foo': '#0'}
         assert expression_attribute_values == {':0': {'S': 'bar'}, ':1': {'S': 'baz'}}
 
+    def test_rand(self):
```

Comment:
Should we also test something like `42 & self.attribute < 'bar'` to exercise the type error branch?